app/src/test/java/woowacourse/study/ReflectionTest.kt

```
@@ -14,7 +14,9 @@ import kotlin.reflect.full.staticFunctions
 
 class Person(var firstName: String, val lastName: String, private var age: Int) {
     fun greeting() {}
+
     private fun fullName() {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)